db/migrate/20150701164051_add_admin_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean, null: false, default: false
```

Comment:
I don't think you have to change this, but for future reference, it's nice to have a `role` attribute for this type of ability. That way you aren't limited to only two roles (admin user and non-admin user), but can have multiple levels user.